server/compaction-coordinator/src/main/java/org/apache/accumulo/coordinator/CompactionCoordinator.java

```diff
@@ -127,7 +127,7 @@
   protected CompactionCoordinator(ServerOpts opts, String[] args) {
     super("compaction-coordinator", opts, args);
     aconf = getConfiguration();
-    schedExecutor = ThreadPools.createGeneralScheduledExecutorService(aconf);
+    schedExecutor = ThreadPools.getServerThreadPools().createGeneralScheduledExecutorService(aconf);
```

Comment:
That would work in the cases where ServerContext is reachable. They are functionally equivalent.

Reply:
> That would work in the cases where ServerContext is reachable. They are functionally equivalent.

While these are functionally equivalent now, it would save effort later, trying to wire in context where needed to get rid of the static references to these resources, in case we wanted to make the server-side UEH configurable at some point.

Reply:
If we can use the context method instead of the static method that would be nice, but that can be follow on work.